frontend/src/components/Csv_Formatter.vue

```diff
@@ -50,10 +18,15 @@ export default {
     },
     data() {
         return {
-           op_response_csv: [], 
+            op_response_csv: [], 
+            total_labels: {}
         }
     },
     methods: {
+        //Create a single csv file with mutliple tables. This is technically not
```

Comment:
CSVs do not allow writing to multiple tabs (like a spreadsheet can), for this reason one large csv file is created with added rows as spacers.